src/annotator/guest.coffee

```diff
@@ -181,7 +181,7 @@ module.exports = class Guest extends Delegator
     this.selections.unsubscribe()
     @adder.remove()
 
-    @element.find('.annotator-hl').each ->
+    @element.find('.hypothesis-highlight').each ->
       $(this).contents().insertBefore(this)
       $(this).remove()
 
```

Comment:
Line 229 and 
Line 329

```
        ignoreSelector: '[class^="annotator-"]'
```

Will that be affected by this? Looks like its got a pattern to ignore "annotator-" but will it also need "hypothesis-" now?

Reply:
Good catch! - I'll update this. We still have CSS classes that use the "annotator-" prefix so we'll need to match both for the time being.